orb/commands/@common.yml

```diff
@@ -203,6 +203,20 @@ helm-release-information:
         command: |
           # Display only the part following NOTES from the helm status.
           helm -n "$NAMESPACE" get notes "$RELEASE_NAME"
+    - run:
+        name: Post release info to Github
+        command: |
+          if [ ! -z $GITHUB_TOKEN ]
+          then
+            RELEASE_NOTES=`helm -n "$NAMESPACE" get notes "$RELEASE_NAME"`
+            if [ -z $CIRCLE_PR_NUMBER ]
```

Comment:
Needs quotes around the tested string (in both comparison statements) and probably double bracketing for if statement

Reply:
Using -z doesn't strictly need quotes like -n does, but you are right it's still better to always use them. Double brackets doesn't change anything functional and wouldn't be Posix compliant.

Reply:
I've seen weird things happen when they are abstracted trough the orb executor, specifically regarding double bracketing, but let's see. If anything we can just fix later.